el-debate/src/main/java/pl/elpassion/elspace/debate/details/DebateDetailsActivity.kt

```diff
@@ -75,36 +78,46 @@ class DebateDetailsActivity : AppCompatActivity(), DebateDetails.View {
     override fun hideLoader() = hideLoader(debateDetailsCoordinator)
 
     override fun showDebateDetailsError(exception: Throwable) {
-        showSnackbar(getString(R.string.debate_details_error))
+        showIndefiniteSnackbar(getString(R.string.debate_details_error))
     }
 
     override fun showVoteError(exception: Throwable) {
-        showSnackbar(getString(R.string.debate_details_vote_error))
+        showIndefiniteSnackbar(getString(R.string.debate_details_vote_error))
     }
 
-    private fun showSnackbar(text: String) {
+    override fun resetImagesInButtons() {
+        debatePositiveAnswerImage.setImageDrawable(ContextCompat.getDrawable(this, R.drawable.hand_positive_inactive))
+        debateNegativeAnswerImage.setImageDrawable(ContextCompat.getDrawable(this, R.drawable.hand_negative_inactive))
+        debateNeutralAnswerImage.setImageDrawable(ContextCompat.getDrawable(this, R.drawable.hand_neutral_inactive))
+    }
+
+    private fun showIndefiniteSnackbar(text: String) {
```

Comment:
You invoke `Snackbar.make()` static method twice in this class so I would rename this method to `showSnackbar()` and pass duration as a 2nd arg with default value.